src/Aspire.Hosting.Azure.Sql/AzureSqlExtensions.cs

```diff
@@ -97,8 +97,9 @@ public static IResourceBuilder<AzureSqlServerResource> AddAzureSqlServer(this ID
     /// <param name="builder">The builder for the Azure SQL resource.</param>
     /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
     /// <param name="databaseName">The name of the database. If not provided, this defaults to the same value as <paramref name="name"/>.</param>
+    /// <param name="skuName">SKU of the database. If not provided, this defaults to the free database tier.<paramref name="name"/>.</param>
     /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
-    public static IResourceBuilder<AzureSqlDatabaseResource> AddDatabase(this IResourceBuilder<AzureSqlServerResource> builder, [ResourceName] string name, string? databaseName = null)
+    public static IResourceBuilder<AzureSqlDatabaseResource> AddDatabase(this IResourceBuilder<AzureSqlServerResource> builder, [ResourceName] string name, string? databaseName = null, string skuName = AzureSqlDatabaseResource.FREE_SKU)
```

Comment:
On the new overload, we shouldn't be hard coding the default SKU in the optional parameter.

```suggestion
    public static IResourceBuilder<AzureSqlDatabaseResource> AddDatabase(this IResourceBuilder<AzureSqlServerResource> builder, [ResourceName] string name, string? databaseName = null, string? skuName = null)
```

And then later in the method body you check if `skuName is null` and use the default.